app/components/Avatar/Avatar.tsx

```diff
@@ -72,4 +67,4 @@ const CircleImg = styled.img<{ size: number; $showBorder?: boolean }>`
   flex-shrink: 0;
 `;
 
-export default Avatar;
+export default observer(Avatar);
```

Comment:
```suggestion
export default Avatar;
```

This shouldn't be needed anymore, only required when the component contains `mobx` state.